### PR TITLE
Add xenial

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

File: 16.04/overlay/etc/apt/apt.conf.d/50unattended-upgrades [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Unattended-Upgrade::Allowed-Origins {
+    "Ubuntu wily-security";
```
Comment (QuentinPerez) [inline 16.04/overlay/etc/apt/apt.conf.d/50unattended-upgrades]: `Ubuntu xenial-security`  ?

Comment (moul) [inline 16.04/overlay/etc/apt/apt.conf.d/50unattended-upgrades]: fixed

**Comment by jaylinski** [comment]
:+1:  don't forget to update the readme

**Comment by moul** [comment]
oh yes, you're right :)

**Pull request merged**